packages/typespec-test/test/NetworkAnalytics.Management/generated/typespec-ts/samples-dev/dataProductsGenerateStorageAccountSasTokenSample.ts

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { NetworkAnalyticsClient } from "@azure/arm-networkanalytics";
+import { DefaultAzureCredential } from "@azure/identity";
+
+/**
+ * This sample demonstrates how to generate sas token for storage account.
+ *
+ * @summary generate sas token for storage account.
+ * x-ms-original-file: 2023-11-15/DataProducts_GenerateStorageAccountSasToken_MaximumSet_Gen.json
+ */
+async function dataProductsGenerateStorageAccountSasTokenMaximumSetGen() {
+  const credential = new DefaultAzureCredential();
+  const subscriptionId = "00000000-0000-0000-0000-00000000000";
+  const client = new NetworkAnalyticsClient(credential, subscriptionId);
+  const result = await client.dataProducts.generateStorageAccountSasToken(
+    "aoiresourceGroupName",
+    "dataproduct01",
+    {
+      startTimeStamp: new Date("2023-08-24T05:34:58.151Z"),
```

Comment:
Is timestamp generated as a date type ?

Reply:
yeah i think so, we have no build tool yet, but i invited jiao confirmed all samples could build compiling.

Reply:
added